packages/Popover/stories/Popover.screener.stories.js

```diff
@@ -17,23 +17,23 @@ storiesOf("Popover/Automation Tests/Screener", module).add("basic", () => (
         <Popover.Card>{shortContent}</Popover.Card>
       </Popover.Content>
     </Popover>
-    <div css={gapStyles} />
+    <div style={gapStyles} />
```

Comment:
Can we add a `Gap` component to the `common.styles.js` instead and avoid inline styling, please.

https://github.com/acl-services/paprika/blob/master/.storybook/assets/styles/common.styles.js